### ISSUE TITLE
Would be nice if each API documentation page had a mechanism for providing feedback

### ISSUE THREAD

**Issue by jamesderlin** [flutter/flutter#22968]
It would be nice if each API documentation page included a link for providing feedback or corrections for the page, similar to MSDN or for what we do for internal g3doc pages. At a minimum it could be a link that starts a GitHub issue with the URL of the page, like: https://github.com/flutter/flutter/issues/new?title=API%20Documentation%20Feedback&body=Feedback%20for%20http%3A%2F%2Fwww.example.com

(Originally filed as https://github.com/flutter/website/issues/1438 )

**Comment by jamesderlin** [comment]
I realized that most of the non-API-documentation pages have "File Issue" links (e.g. https://flutter.io/widgets/), so the API documentation pages should do the same for consistency.

**Comment by InMatrix** [comment]
This feature is currently blocked by https://github.com/dart-lang/dartdoc/issues/1454. Please consider upvoting that issue.

**Comment by InMatrix** [comment]
> As a first step, being able to file a GitHub issues that includes the URL to the documentation page itself would be an improvement. 

Agreed. I wonder if this requires changes to DartDoc. Cc: @gspencergoog @jcollins-g

**Comment by jcollins-g** [comment]
Filed as dart-lang/dartdoc#1815.

**Comment by SogoGolf** [comment]
i was just about to file a ticket and saw this. 

Microsoft Azure do this really well with a "Feedback" widget at bottom of each developer docs page. What I really like about that is that it keeps the developer (me!) right in the context of what I'm doing/looking for - rather than having to go across to Github, maybe login, find the right repo, open a ticket,..etc etc etc

Here is a screenshot of what I'm referring to on Microsoft Azure. Something similar would be brilliant and super helpful for Flutter.

![image](https://user-images.githubusercontent.com/19580787/50474270-6dd7e580-0a14-11e9-8375-cde7b6cb9d86.png)
